app/src/main/java/com/example/playlistmaker/domain/search/api/GetJsonFromTrackUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.playlistmaker.domain.search.api
+
+import com.example.playlistmaker.domain.models.Track
+
+interface GetJsonFromTrackUseCase {
```

Comment:
Все интерфейсы с одной функцией внутри можно пометить ключевым словом `fun`, чтобы сделать интерфейс функциональным:
```suggestion
fun interface GetJsonFromTrackUseCase {
```